packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -480,13 +480,12 @@ class CameraController extends ValueNotifier<CameraValue> {
   /// Throws a [CameraException] if image streaming or video recording has
   /// already started.
   ///
-  /// The `startImageStream` method is only available on Android and iOS (other
-  /// platforms won't be supported in current setup).
+  /// The `startImageStream` method is only available on platforms that
+  /// report support for image streaming via [CameraPlatform.supportsImageStreaming].
```

Comment:
This needs to be wrapped in a method at this layer; clients are not expected to use the platform interface directly.

Reply:
Added a method.